pyproject.toml

```diff
@@ -1,3 +1,32 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
```

Comment:
```suggestion
```

I _think_ we can omit this entirely then?

Reply:
TIL!